public/javascript/script.js

```diff
@@ -0,0 +1,90 @@
+const video = document.getElementById('videoInput')
+
+Promise.all([
+    faceapi.nets.faceRecognitionNet.loadFromUri('/models'),
+    faceapi.nets.faceLandmark68Net.loadFromUri('/models'),
+    faceapi.nets.ssdMobilenetv1.loadFromUri('/models') //heavier/accurate version of tiny face detector
+]).then(start)
+
+function start() {
+    document.body.append('Models Loaded')
+
+    navigator.getUserMedia(
```

Comment:
This method is deprecated. Use the newer method. Read more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia).
```suggestion
    navigator.mediaDevices.getUserMedia(
```